src/routes/LoginPrompt/index.jsx

```diff
@@ -27,7 +27,7 @@ const LoginPrompt = ({
 
   useEffect(() => {
     if (isLoggedIn) {
-      navigate(nextPageUrl || "/self-service/branding");
+      navigate(nextPageUrl || "/work/dashboard");
```

Comment:
plz use the global constant for this route:  ROUTES.DASHBOARD_PAGE

Reply:
changed 👍🏻